### ISSUE TITLE
UI - Add ability to delete experiments

### ISSUE THREAD

**Issue by gtarcea** [materials-commons/materialscommons.org#986]
Allow users to delete experiments

**Comment by Terry-Weymouth** [comment]
Depends on #1015 (backend - ability to delete experiment)

**Comment by gtarcea** [comment]
Until this is fixed the work around is to mark the experiment as on hold or completed.
